Resources/Prototypes/_SpaceCats/ai_factions.yml

```
@@ -0,0 +1,37 @@
+  - Xeno
+  - Zombie
+  - Revolutionary
+  - Blob
+  - SuperZloy
+  - Spanish
+  - Britan
+  - SpecialForce
+  - CombineForce
+  - CentralCommand
+  - FleshHuman
+  - Flesh
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)